src/main/java/pl/tomaszdziurko/jvm_bloggers/SpringApplicationContext.java

```diff
@@ -0,0 +1,21 @@
+package pl.tomaszdziurko.jvm_bloggers;
+
+import org.springframework.beans.BeansException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SpringApplicationContext implements ApplicationContextAware {
```

Comment:
You can just Di ApplicationContext using @Autowired ...implementing interface is not needed..or you vae good reasons to do that ;]